test/Spec.hs

```diff
@@ -119,6 +120,9 @@ tests docker0 = testGroup "Tests"
       , testProperty "3-threadsEquivFailingNoOp"
         $ expectFailure $ withMaxSuccess 1000 $ prop_nparallel_clean  3 (Eq True)  Cleanup.NoBug     NoOp
       ]
+  , testGroup "SQLite"
+      [ testProperty "Parallel" prop_parallel_sqlite
```

Comment:
The sequential property is missing here.

Reply:
That's intentianal. When I ran it localy, leaving one property fixed the issue with runMigrationSilent. I guessed this happened because Tasty runs properties in parallel. But in ci it failed even with one property :confused:

Reply:
What happens if you use `runMigration` rather than `runMigrationSilent` (that seems to work in the CRUD example)?

Reply:
It floods the test output, since I run the Migration on every test.

Reply:
I changed to runMigration and fixed some other warnings.